scripts/updatedb.js

```diff
@@ -24,21 +38,34 @@ const Address6 = require('ip-address').Address6
 const Address4 = require('ip-address').Address4
 
 const args = process.argv.slice(2)
-let licenseKey = args.find(function (arg) {
+let license_key = args.find(function (arg) {
```

Comment:
hmm maybe using yargs is safer here since we're already refactoring this?

Reply:
i haven't refactored at all, only refreshed the script from upsgream.    Maxmind downloads have been fragile and we have to merge this script from upstream sort of often.  

In the past I changed to camelcase...but this time to minimixe changes, I turned off lint from complaining about it.

Reply:
so we are now much closer to the upstream script